src/components/WheelPicker/index.tsx

```diff
@@ -200,9 +205,9 @@ const WheelPicker = ({
     setTimeout(() => scrollToOffset(index, animated), 100);
   };
 
-  const scrollToPassedIndex = useCallback(() => {
-    scrollToIndex(currentIndex, false);
-  }, []);
+  // const scrollToPassedIndex = useCallback(() => {
```

Comment:
Why to keep it commented?

Reply:
I thought it will be good to keep it comment (as documentation for us) and the `onLayout` prop in the `FlatList` in case of any issue with it.
Should I delete it?

Reply:
I don't see a need to keep it. If the fix will cause issues we can revert it.